tests/baselines/reference/api/typescript.d.ts

```diff
@@ -9908,19 +9917,27 @@ declare namespace ts {
      * @param file File to fetch the resolution mode within
      * @param index Index into the file's complete resolution list to get the resolution of - this is a concatenation of the file's imports and module augmentations
      */
-    function getModeForResolutionAtIndex(file: SourceFile, index: number): ResolutionMode;
+    function getModeForResolutionAtIndex(file: SourceFile, index: number, compilerOptions: CompilerOptions): ResolutionMode;
```

Comment:
This is a breaking change; maybe it was discussed somewhere else in the PR (I can't find it), but is this intentional? It does appear to be used externally. https://github.com/search?q=%2F%5C.getModeForResolutionAtIndex%2F&type=code